src/eureka/S4_generate_lightcurves/s4_genLC.py

```diff
@@ -294,6 +310,10 @@ def genlc(eventlabel, ecf_path=None, s3_meta=None):
                     plots_s4.driftxpos(meta, lc)
                     plots_s4.driftxwidth(meta, lc)
 
+            if hasattr(meta, 'manual_clip') and meta.manual_clip is not None:
```

Comment:
Manual clipping should go into Stage 5.  When fitting the LCs, people will want to try different clip sizes without having to rerun S4.